AirshipDemo/src/CustomFloatingComponent.tsx

```diff
@@ -11,8 +11,8 @@ interface Props {
  */
 export function CustomFloatingComponent(props: Props): JSX.Element {
   const { bridge } = props
-  React.useEffect(() => bridge.on('clear', bridge.resolve), [])
-  React.useEffect(() => bridge.on('result', bridge.remove), [])
+  React.useEffect(() => bridge.on('clear', bridge.resolve), [bridge])
```

Comment:
Are you sure this is just a "lint" fix?
It does seem to change the logic of when to call the effect.

Reply:
The bridge never changes, so the effect will run once & never again, just as before. But now the linter is happy because we put all the dependencies in the list.

Reply:
Ok. As long as you're sure that the bridge won't change then this is approved.